include/mamba/core/version.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019, QuantStack and Mamba Contributors
```

Comment:
this file is generated from version.hpp.in and shouldn't be commited

Reply:
Nice catch, sorry for that. It's fixed

Reply:
I think this pattern should be reversed at some point (read version.hpp form CMake instead of generating the code with CMake). It is usually considered as a bad practice to generate code from your build system since it makes it difficult for someone to build the sources with a different build system.